src/akamai/js/akamaiHD_flash.js

```diff
@@ -311,6 +311,21 @@ require("../../../html5-common/js/utils/constants.js");
      * @method OoyalaAkamaiHDFlashVideoWrapper#destroy
      */
     this.destroy = function() {
+      // Pause the video
+      this.pause();
+      
+      // Reset the source
+      this.setVideoUrl('');
```

Comment:
not sure if you need to do this for flash - we were just doing this for html5 proper video element cleanup